pkg/blobstore/local/flat_blob_access.go

```diff
@@ -361,6 +409,7 @@ func (ba *flatBlobAccess) FindMissing(ctx context.Context, digests digest.Set) (
 				// Blob is present and still needs to be
 				// refreshed. Allocate space for a copy.
 				b := getter(blobToRefresh.digest)
+				ba.refreshSizeFindMissing.Observe(float64(location.SizeBytes))
```

Comment:
I think it would be more valuable to aggregate the sizes and report it for the refresh as a whole, because then these metrics can be used together with the other metrics we have for refreshing. Now the number of samples within will differ.

Reply:
done.